### PR TITLE
Updating CC list

### PR DESCRIPTION
**Pull request by pscharla**
Updating the CC list in our docs. @RackHD/corecommitters

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -34,10 +34,13 @@ If you have questions about the disposition of a request, feel free to email one
...
-* Felix.Yuan@emc.com
-* Andrew.Hou@emc.com
-* Brian.Parry@emc.com
+* Jeanne.Ohren@dell.com
+* Felix.Yuan@dell.com
+* Andrew.Hou@dell.com
+* Brian.Parry@dell.com
+* Rahman.Muhammad@dell.com
+* Geoffrey.Reid@dell.com
+
```
Comment (johren) [inline docs/contributing.rst]: I believe James.King@dell.com also needs to be added.

File: docs/contributing.rst [hunk 2]
```diff
@@ -34,10 +34,13 @@ If you have questions about the disposition of a request, feel free to email one
...
 
-* Ben.BroderickPhillips@emc.com
-* Felix.Yuan@emc.com
-* Andrew.Hou@emc.com
-* Brian.Parry@emc.com
+* Jeanne.Ohren@dell.com
+* Felix.Yuan@dell.com
+* Andrew.Hou@dell.com
+* Brian.Parry@dell.com
+* Rahman.Muhammad@dell.com
```
Comment (johren) [inline docs/contributing.rst]: I have Rahman_Muhammad@dell.com in my emails.  Unless there are multiple aliases.

**Comment by johren** [comment]
Thanks for updating, Paul!

File: docs/contributing.rst (hunk 1 above)
Comment (pscharla) [inline docs/contributing.rst]: Good call. Thanks!

File: docs/contributing.rst (hunk 2 above)
Comment (pscharla) [inline docs/contributing.rst]: Maybe - I'll check with him

Comment (pscharla) [inline docs/contributing.rst]: He confirmed: Rahman.Muhammad@dell.com works

Comment (johren) [inline docs/contributing.rst]: 👍

**Pull request merged**